src/components/Jobs/jobs.util.js

```diff
@@ -43,13 +43,17 @@ import { showErrorNotification } from '../../utils/notifications.util'
 
 export const page = JOBS_PAGE
 const LOG_LEVEL_ID = 'logLevel'
-export const getInfoHeaders = isSpark => {
+export const getInfoHeaders = (isSpark, selectedJob) => {
   const infoHeaders = [
     { label: 'UID', id: 'uid' },
     { label: 'Start time', id: 'startTime' },
     { label: 'Last Updated', id: 'updated' },
     { label: 'Run on spot', id: 'runOnSpot' },
-    { label: 'Node selector', id: 'nodeSelectorChips' },
+    {
+      label: 'Node selector',
+      id: 'nodeSelectorChips',
+      hidden: isJobKindDask(selectedJob?.labels) || !selectedJob?.nodeSelectorChips?.length
+    },
```

Comment:
I don't think we need to hide any field if its value is empty. We don't do that for other fields.

Reply:
@mavdryk that is the bug. hide the field when function is dask or no NS. 
What do you suggest?

Reply:
@ilan7empest Yes, we must hide if the function is dask, but why should we hide if there is no NS? We don't hide other fields if their value is empty.